docs/application-guide/trace-explorer.md

```diff
@@ -0,0 +1,119 @@
+Explorer for your AWS Lambda functions when [Instrumentation](./instrumentation.md)
+is enabled.
+
+Serverless Console provides a set of tools to analyzing Traces.
+
+## Trace Explorer List
+
+Similar to the [Metrics View](./metrics.md), the Trace Explorer provides a
+starting point for troubleshooting AWS Lambda function invocations across your
+org. You can use the rich filters to narrow in on errors, warnings, and
+performance issues across all of your AWS Lambda functions across your org.
+
+## Filters
+
+Filtering allows you to narrow in on particular behavior and time frame for 
+to isolate invocations. You can filter on:
+
+- **Event Types** - Errors and Warnings can be captured in the trace, these
+includes user defined as well as SDK defined errors and warnings. More details
+on each Event type is available below.
+- **Event Messages** - When an Event like an error or warning is captured, a
+message string is saved with the Event. You can filter for the Traces based on
+the Event messages that were captured in the trace. Traces are filtered if any
+of the Events in the Trace contained the message string.
+- **Resource** - You can select the specific resource by AWS ARN, like a
+specific Lambda function.
+- **Environment**, **Namespace** - These properties are inferred from the
+CloudFormation stack when Instrumentation is added, or they are set manually
+on the Integration settings page. Once set, you can filter the traces based on
+these properties set on the function.
+- **AWS Account**, **Region** - Serverless Console collects information for all
+instrumented Lambda functions across AWS accounts and regions; you  can filter
+on any of these properties.
+- **Timeframe** - Any timeframe within the last 30 days can be used.
+
+## Event Types
+
+- `ERROR_TYPE_UNCAUGHT` - The Lambda function handler had a fatal error and
+caused the invocation to fail.
+- `ERROR_TYPE_CAUGHT_USER` - The Lambda function handler had an error that was
+captured using the SDK. This can include Error objects or strings that were
+captured implicitly (e.g. `console.error`) or explicitly using the provided
+SDK methods.
+- `ERROR_TYPE_CAUGHT_SDK_USER` - An internal SDK error that was reported due to
+misuse of the SDK. These errors do not cause handler failures, but misusage of
+the SDK may result in partial collection. For example, using the `setTag` method
+with invalid inputs will result in this type of error, and the tag will not be
+set.
+- `ERROR_TYPE_CAUGHT_SDK_INTERNAL` - This is an internal SDK error. These errors
+will not cause handler failures. This type of error is rare and monitored by
+Serverless.
+- `WARNING_TYPE_USER` - This is a warning message that was generated in the
+handler and collected by the SDK. For example, `console.warn("record lookup
+failed")`, would result in this type of warning.
+- `WARNING_TYPE_SDK_USER` - This is an internal SDK warning that was reported
+due to misuse of the SDK.
```

Comment:
This is a warning reported internally by the SDK, but which signals not SDK related potential problem

e.g. currently we issue that warning if we find the AWS SDK v2 requests configuration, where both callback and promise resolution is attached (in such case AWS SDK issues two requests underneath, and that may produce unwanted side-effects)

Reply:
To be fully clear, `_SDK_` in that code means that a given event is implicitly reported by the SDK and not explicitly by the user.

The event may be related to SDK usage, but doesn't have to, e.g. in the case of `WARNING_TYPE_SDK_USER`, so far configured events report purely on user potential misusage of AWS SDK (not our Console SDK)